Services/Cache.cs

```diff
@@ -154,7 +226,7 @@ public class Cache : ICache
                 }
                 catch (ResourceNotFoundException)
                 {
-                    this.log.Info($"{CACHE_COLLECTION_ID}:{CACHE_KEY} not found.", () => {});
+                    this.log.Info($"Cache updating: cache {CACHE_COLLECTION_ID}:{CACHE_KEY} was not found", () => { });
```

Comment:
error?

Reply:
It is not an error. It will always happen when the cache was "rebuilding" for the first time, since there is no such an item in the storage.